naga-cli/src/bin/naga.rs

```diff
@@ -42,7 +42,13 @@ struct Args {
     #[argh(option)]
     block_ctx_dir: Option<String>,
 
-    /// the shader entrypoint to use when compiling to GLSL
+    /// the shader entrypoint.
+    ///
+    /// This is used by the GLSL and SPIR-V backends.
+    ///
+    /// When `--compact` is specified, the entrypoint is passed to
+    /// `process_overrides`, and anything not reachable from the entry point will
```

Comment:
suggestion: `process_overrides` is not a concept I expect most CLI users to know about. If they don't know the name already, it also doesn't really help them get the mental model they're missing. It doesn't seem really necessary to mention it, so I suggest taking out the reference to it and either simply summarizing:

```suggestion
    /// When `--compact` is specified, anything not reachable from the entry point will
```

…modulo new line breaks, or maybe adding a bit more as a subsequent paragraph if you think it's helpful.